app/js/checkAnswers.ts

```diff
@@ -59,7 +59,10 @@ async function checkAnswers(userAnswers: UserAnswers): Promise<any> {
  */
 async function getAnswers() {
     let baseUrl = getBaseUrl();
-    let data = await fetch(baseUrl + "answer", {method: 'get'});
+    let cookie = getCookie('userEmail');
+    let userData = await getData(`user?email=${cookie}`)
```

Comment:
I have seen some userData somewhere (I think poss only in loops and stuff but maybe have a look to see if there are any that might crossover